techniques/general/G209.html

```diff
@@ -146,37 +142,37 @@ <h3>Example 4: Pie chart with black and white borders</h3>
     
 		
         <section id="resources">
-                <h2>Related Resources</h2>
-<p style="margin-bottom: 1.5em;"><em><small>No endorsement implied.</small></em></p>
+                <h2>関連リソース</h2>
+<p style="margin-bottom: 1.5em;"><em><small>推奨を意味するものではない。</small></em></p>
 
 
                 <ul>
-                    <li><a href="https://w3c.github.io/wcag/understanding/contrast-minimum.html#resources">Understanding SC 1.4.3 Contrast (minimum)</a> lists color contrast tools.</li>
+                    <li><a href="https://waic.jp/translations/WCAG22/Understanding/contrast-minimum.html#resources">解説書 達成基準 1.4.3 コントラスト (最低限)</a> にあるカラーコントラストツール</li>
```

Comment:
G207 にも同じ文面がありますが、そこでは以下のように訳しています。

> 解説書 達成基準 1.4.3 コントラスト (最低限) はカラーコントラストツールを挙げている。

lists を動詞と捉えるならば、このほうがよいかなと思います。

```suggestion
                    <li><a href="https://waic.jp/translations/WCAG22/Understanding/contrast-minimum.html#resources">解説書 達成基準 1.4.3 コントラスト (最低限)</a> はカラーコントラストツールを挙げている。</li>
```